flusher.go

```diff
@@ -64,9 +67,7 @@ func (s *Server) Flush(interval time.Duration, metricLimit int) {
 		// local-only histograms will be flushed with percentiles, so we intentionally
 		// use the original percentile list here
 		totalLocalSets + (totalLocalTimers+totalLocalHistograms)*(HistogramLocalLength+len(s.HistogramPercentiles))
-	if s.ForwardAddr == "" {
-		totalLength += totalSets
-	}
+	totalLength += totalSets
```

Comment:
I like this change! I recall when this code first appeared feeling odd about his method for deciding what we were doing. :)